exercises/practice/nth-prime/.meta/example.py

```diff
@@ -4,7 +4,7 @@
 
 def prime(number):
     if number < 1:
-        raise ValueError('The parameter `number` has to be a positive number.')
+        raise ValueError('there is no zeroth prime')
```

Comment:
I mean I know this is an exemplar, but is `zeroth` really _that_ clear? :)

Reply:
You could always take it up in `problem-specifications`.  That wording came straight from the [`canonical.json`](https://github.com/exercism/problem-specifications/blob/00f45ee2f7666ab5d33f1ea18ae9b9a4db169821/exercises/nth-prime/canonical-data.json#L48).  😉

Reply:
Ah alrighty!